serverless-now/env/sandbox.json

```diff
@@ -0,0 +1,12 @@
+{
+    "SERVICENOW_BASE_URL": "api.dev.auckland.ac.nz",
+    "COGNITO_DOMAIN": "uoapool-sandbox.auth.ap-southeast-2.amazoncognito.com",
+    "EXAMPLE_KEY": "${ssm:/sandbox/research-hub/example-key~true}",
+    "SN_API_KEY_R": "${ssm:/sandbox/research-hub/SN_API_KEY_R~true}",
+    "SN_DEV_API_KEY_RW": "${ssm:/sandbox/research-hub/SN_DEV_API_KEY_RW~true}",
+    "OAUTH_SCOPES": [
+        "openid",
+        "profile",
+        "https://test-domain.auckland.ac.nz/lambda-hello-world"
```

Comment:
Can you check if it still works if you remove this third `OATH_SCOPE`, it was just enabled while we were setting up Cognito but we will probably end up removing it soon. If it breaks things no worries we can remove it later :v:

Reply:
Yep sure. Seems to pass unit tests and postman tests just fine so I'll remove it.